### PR TITLE
fix: added blurMultiplier adjustment

### PR DESCRIPTION
**Pull request by xionglinlin**
The blur effect can be adjusted by blurMultiplier

Issue: https://github.com/linuxdeepin/developer-center/issues/9374

### CODE REVIEW

**Review by 18202781743 — APPROVED** [review]

**Comment by hudeng-go** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dtkdeclarative/pull/352#pullrequestreview-2131755519" title="Approved">18202781743</a>*, *<a href="https://github.com/linuxdeepin/dtkdeclarative/pull/352#" title="Author self-approved">xionglinlin</a>*, *<a href="https://github.com/linuxdeepin/dtkdeclarative/pull/352#pullrequestreview-2132217707" title="Approved">zccrs</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdtkdeclarative).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dtkdeclarative/blob/master/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
